control_panel_api/serializers.py

```diff
@@ -24,10 +24,12 @@ class Meta:
 
 
 class AppSerializer(serializers.ModelSerializer):
-
     class Meta:
         model = App
-        fields = ('id', 'url', 'name', 'slug', 'repo_url', 'apps3buckets')
+        fields = (
+            'id', 'url', 'name', 'slug', 'repo_url', 'apps3buckets',
```

Comment:
Minor formatting point - it's preferable to have one item per line, so that when the list changes, diffs are clearer.

Reply:
I tend to prefer one per line for "long" lists as well. Should we have a "rule" 🚔  for if something has more than 3 items split it straight away? (with trailing comma on the last element :trollface: )

Reply:
Updated for the diff.